ibm_catalog.json

```diff
@@ -134,42 +134,76 @@
               "on_by_default": true
             },
             {
-              "name": "deploy-arch-ibm-observability",
-              "description": "Enable to provision and configure IBM Cloud Logs, Cloud Monitoring, Metrics routing and Activity Tracker event routing for analysing logs and metrics generated by the SCC Workload Protection instance.",
+              "name": "deploy-arch-ibm-cloud-logs",
+              "description": "Provision and configure IBM Cloud Logs (ICL) instance which can be used for analysing logs generated by SCC Workload Protection instance. ",
+              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
               "flavors": [
-                "instances"
+                "fully-configurable"
               ],
-              "id": "a3137d28-79e0-479d-8a24-758ebd5a0eab-global",
-              "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+              "id": "63d8ae58-fbf3-41ce-b844-0fb5b85882ab-global",
+              "version": "v1.6.11",
+              "optional": true,
+              "on_by_default": true,
               "input_mapping": [
-                {
-                  "dependency_output": "cloud_monitoring_crn",
-                  "version_input": "existing_monitoring_crn"
-                },
                 {
                   "dependency_input": "prefix",
                   "version_input": "prefix",
                   "reference_version": true
                 },
                 {
-                  "dependency_input": "region",
-                  "version_input": "region",
+                  "dependency_input": "existing_resource_group_name",
+                  "version_input": "existing_resource_group_name",
                   "reference_version": true
                 },
                 {
-                  "dependency_input": "enable_platform_metrics",
-                  "version_input": "enable_platform_metrics",
+                  "dependency_input": "region",
+                  "version_input": "region",
                   "reference_version": true
                 },
                 {
                   "dependency_input": "logs_routing_tenant_regions",
                   "version_input": "logs_routing_tenant_regions",
                   "reference_version": true
                 }
+              ]
+            },
+            {
+              "name": "deploy-arch-ibm-cloud-monitoring",
+              "description": "Provision and configure IBM Cloud Monitoring for metrics generated by the SCC Workload Protection instance.",
```

Comment:
Can you be consistent with what was used in https://github.com/terraform-ibm-modules/terraform-ibm-app-configuration/pull/269 ? We want to try to keep this description short and clear